src/index.js

```diff
@@ -0,0 +1,157 @@
+    state.resetButton = document.getElementById("cityNameReset");
+    state.skySelect = document.getElementById("skySelect");
+    state.sky = document.getElementById("sky");
+};
+
+const increaseTemp = () => {
+    state.tempDeg += 1;
+    tempValue.textContent = `${state.tempDeg}`
+    changeColor();
+    changeLandscape();
+};
+
+const decreaseTemp = () => {
+    state.tempDeg -= 1;
+    tempValue.textContent = `${state.tempDeg}`;
+    changeColor();
+    changeLandscape();
+};
+
+const changeColor = () => {
+    let currentTemp = tempValue.textContent;
+    if (currentTemp >= 80) {
+        tempValue.style.color = "red";
+    } else if (currentTemp >= 70) {
+        tempValue.style.color = "orange";
+    } else if (currentTemp >= 60) {
+        tempValue.style.color = "yellow";
+    } else if (currentTemp >= 50) {
+        tempValue.style.color = "green";
+    } else {
+        tempValue.style.color = "teal";
+    }
+};
+
+const changeLandscape = () => {
+    let currentTemp = tempValue.textContent;
+    if (currentTemp >= 80) {
+        landscape.textContent = `${state.hotLandscape}`;
+    } else if (currentTemp >= 70) {
+        landscape.textContent = `${state.niceLandscape}`;
+    } else if (currentTemp >= 60) {
+        landscape.textContent = `${state.midLandscape}`;
+    } else {
+        landscape.textContent = `${state.defaultLandscape}`;
+    }
+};
+
+const displayCityName = (event) => {
+    const cityInput = event.target.value; 
+    state.cityNameOutput.textContent = cityInput;
+};
+
+const changeSky = () => {
+    sky.textContent = `${skySelect.options[skySelect.selectedIndex].text}`;
+}
+
+const displayCurrentWeather = () => {
+    findLatAndLon();
+}
+
+const findLatAndLon= (query) => {
+    let latitude, longitude;
+    axios.get('http://localhost:5000/location',
+    {
+        params: {
+            q: `${state.cityNameOutput.textContent}`,
+            format: 'json'
+        }
+})
+.then( (response) => {
+    latitude = response.data[0].lat;
+    longitude = response.data[0].lon;
+    console.log('success in findLatAndLon', latitude, longitude)
+        
+    findWeather(latitude, longitude); 
+})
+.catch( (error) => {
+    console.log('error in findLatAndLon!');
+})
+};
+
+const findWeather = (latitude, longitude) => {
+    axios.get('http://localhost:5000/weather',
+    {
+        params: {
+            format: 'json',
+            lat: latitude,
+            lon: longitude
+        }
+    })
+    .then( (response) => {
+    
+    console.log('success in findWeather!', response.data.main.temp);
+    state.tempDeg = Math.round((response.data.main.temp - 273.15) * 9/5 + 32);
+    tempValue.textContent = `${state.tempDeg}`
+    changeColor();
+    changeLandscape();
+    })
+    .catch( (error) => {
+        console.log('error in findWeather!');
+    });
+}
+
+const onLoaded = () => {
+    loadControls();
+    setDefaultValues();
+    registerEventHandlers();
+};
+
+onLoaded();
```

Comment:
We had to use the onLoaded approach when we were using codesandbox because by the time the UI had rendered and shown us the code editor for to write code the event called DOMContentLoaded had already fired. The act of rendering the web page makes DOMContentLoaded happen so we couldn't use it as an event to register loadControls(). 

When you're writing vanilla JS for event handling like, we should call register the function called `registerEventHandlers` on the event DOMContentLoaded, like this: 
```javascript
window.addEventListener("DOMContentLoaded", registerEventHandlers)
``` 

Then when your page loads `registerEventHandlers` will execute. 

You could add `loadControls()` and `setDefaultValues() inside `registerEventHandlers()` so alll your 'start up' logic happens on page load.